src/views/GovernanceReviewTeam/Dates/index.tsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useParams } from 'react-router-dom';
+import { Button } from '@trussworks/react-uswds';
+import { Field, Form, Formik, FormikProps } from 'formik';
+import { DateTime } from 'luxon';
+
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import Label from 'components/shared/Label';
+import TextField from 'components/shared/TextField';
+import { AppState } from 'reducers/rootReducer';
+import { saveSystemIntake } from 'types/routines';
+import { SubmitDatesForm, SystemIntakeForm } from 'types/systemIntake';
+import flattenErrors from 'utils/flattenErrors';
+
+const Dates = ({ systemIntake }: { systemIntake: SystemIntakeForm }) => {
+  const { systemId } = useParams();
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const { t } = useTranslation();
+  const intakeIsLoading = useSelector(
+    (state: AppState) => state.systemIntake.isLoading
+  );
+
+  const { grtDate, grbDate } = systemIntake;
+
+  // TODO: Fix Text Field so we don't have to set initial empty values
+  const initialValues: SubmitDatesForm = {
+    grtDateDay: grtDate ? String(grtDate.day) : '',
```

Comment:
```suggestion
    grtDateDay: String(grtDate.day) || '',
```

I think that'd work. Reads a little easier to me, but this suggestion isn't required.

Reply:
So, typescript yells at this because grtDate is potentially nil, and therefore doesn't have a `day` property. And typescript is right in this case. :-)